release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -570,6 +570,12 @@ Beginning with {product-title} {product-version}, you can upgrade or downgrade t
 
 In {product-title} {product-version} and later, you can edit the baseboard management controller (BMC) address in the `BareMetalHost` resource of a bare-metal node. The node must be in the `Provisioned`, `ExternallyProvisioned`, `Registering`, or `Available` state. Editing the BMC address in the `BareMetalHost` resource will not result in deprovisioning the node. See xref:../post_installation_configuration/bare-metal-configuration.adoc#editing-a-baremetalhost-resource_post-install-bare-metal-configuration[Editing a BareMetalHost resource] for additional details.
 
+[id="ocp-4-16-attaching-a-non-bootable-iso_{context}"]
+==== Attaching a non-bootable ISO
+
+In {product-title} {product-version} and later, you can attach a generic, non-bootable ISO virtual media image to a provisioned node by using the `DataImage` resource. After you apply the resource, the ISO image becomes accessible to the operating system on the next reboot.
+The node must use Redfish or drivers derived from it to support this feature. The node must be in the `Provisioned` or `ExternallyProvisioned` state. See xref:../post_installation_configuration/bare-metal-configuration.adoc#attaching-a-non-bootable-iso-to-a-bare-metal-node_post-install-bare-metal-configuration[Attaching a non-bootable ISO to a bare-metal node] for additional details.
```

Comment:
The links mentioned in this reference don't exist. For non-bootable iso it might be better to directly use the link : https://github.com/openshift/openshift-docs/blob/enterprise-4.16/modules/bmo-attaching-a-non-bootable-iso-to-a-bare-metal-node.adoc  ( perhaps change enterprise-4.16 with {product-version} ? ).

Reply:
The reference does exist. The link has been tested and it does work. We use xrefs, not fully qualified links, because they get published in both docs.openshift.com and docs.redhat.com. The link in the preview at https://77822--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-attaching-a-non-bootable-iso_release-notes navigates successfully to https://77822--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/bare-metal-configuration#attaching-a-non-bootable-iso-to-a-bare-metal-node_post-install-bare-metal-configuration.